### PR TITLE
docs: update showcase images; remove GraphQL Code Generator site

### PR DESCRIPTION
**Pull request by PatelN123**
## Motivation

My motivation to do this was to be able to see everyone's creations, whilst also doing something useful!

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

---

I have also spotted a few other issues, but they'll be added in a second.

### CODE REVIEW

**Comment by PatelN123** [comment]
I suspect the site https://graphql-code-generator.com/ is not using docusaurus. Please confirm this @Josh-Cena

**Comment by Josh-Cena** [comment]
Yep, can confirm that GCG is now using Next.js

**Comment by PatelN123** [comment]
Thanks, did suspect it but I figured to leave it to the end just in case I'm just blind 🤣

**Comment by PatelN123** [comment]
I should also mention that this reduces the file size of some of the images. :)

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -180,14 +180,6 @@ const Users: User[] = [
     source: 'https://github.com/algolia/docsearch-website',
     tags: ['favorite', 'opensource', 'product'],
   },
-  {
-    title: 'Amphora Data',
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:184]: FWIW, I'd rather not remove sites that just happen to be down unless for good reason. At least we do have the repo available. Could you file an issue in their repo asking about their site's status, revert the removal here, and just add a `// TODO: site unavailable on 12/31/2021`?

**Comment by Josh-Cena** [comment]
> I should also mention that this reduces the file size of some of the images. :)

Yeah, we also have #6043 to resize the images because a lot of contributors submitted 2880px images...

File: website/src/data/users.tsx (hunk 1 above)
Comment (PatelN123) [reply website/src/data/users.tsx:184]: Sure, gimmie a sec.

File: website/src/data/users.tsx [hunk 2]
```diff
@@ -180,14 +180,6 @@ const Users: User[] = [
...
-    description: 'Connecting the world’s real-time information',
-    preview: require('./showcase/amphora.png'),
-    website: 'https://www.amphoradata.com/',
-    source: 'https://github.com/amphoradata/amphoradata.github.io',
```
Comment (PatelN123) [inline website/src/data/users.tsx:188]: ✔️ Done
https://github.com/amphoradata/amphoradata.github.io/issues/44

File: website/src/data/users.tsx [hunk 3]
```diff
@@ -232,14 +224,6 @@ const Users: User[] = [
     source: 'https://gitlab.com/aweframework/awe',
     tags: ['opensource', 'i18n', 'versioning'],
   },
-  {
-    title: 'Axioms',
-    description: 'Axioms Developer Hub and Documentation Portal',
-    preview: require('./showcase/axioms.png'),
-    website: 'https://developer.axioms.io/',
-    source: 'https://github.com/axioms-io/developer',
```
Comment (PatelN123) [inline website/src/data/users.tsx:240]: ✔️ Done
https://github.com/axioms-io/developer/issues/1

File: website/src/data/users.tsx [hunk 4]
```diff
@@ -960,14 +944,6 @@ const Users: User[] = [
     source: 'https://github.com/orbitjs/orbit/tree/main/website',
     tags: ['opensource', 'versioning'],
   },
-  {
-    title: 'Oxidizer',
-    description: 'A Rust ORM based on tokio-postgres and refinery',
-    preview: require('./showcase/oxidizer.png'),
-    website: 'https://oxidizer.rs',
-    source: 'https://github.com/oxidizer-rs/website',
```
Comment (PatelN123) [inline website/src/data/users.tsx:968]: ✔️ Done
https://github.com/oxidizer-rs/website/issues/163

File: website/src/data/users.tsx [hunk 5]
```diff
@@ -1576,14 +1552,6 @@ const Users: User[] = [
     source: 'https://github.com/webiny/docs.webiny.com',
     tags: ['opensource', 'product'],
   },
-  {
-    title: 'Wisdom',
-    description: 'Session replay web analytics with open data SQL/S3 access.',
-    preview: require('./showcase/wisdom.png'),
-    website: 'https://developers.getwisdom.io/',
-    source: 'https://github.com/Wisdom/dev-docs',
```
Comment (PatelN123) [inline website/src/data/users.tsx:1584]: ✔️ Done
https://github.com/Wisdom/dev-docs/issues/1

File: website/src/data/users.tsx (hunk 1 above)
Comment (PatelN123) [reply website/src/data/users.tsx:184]: Done

**Comment by PatelN123** [comment]
Reverted

**Comment by Josh-Cena** [comment]
> just add a `// TODO: site unavailable on 12/31/2021`?

🤣 I was asking if you could add this at the top of those unavailable entries, because `TODO` directives will be tracked by my editor so I can easily find them

**Comment by PatelN123** [comment]
Oh right. Edited them 👌🏽 Is that alright now @Josh-Cena?

**Comment by Josh-Cena** [comment]
Did you push?

**Comment by PatelN123** [comment]
Did I push what? Sorry.

**Comment by Josh-Cena** [comment]
Don't see the latest commit to add the `TODO` comments...

**Comment by PatelN123** [comment]
I have a feeling I am not doing the TODO correctly. 

image

**Comment by Josh-Cena** [comment]
Never mind, I've done it myself.

**Comment by Josh-Cena** [comment]
Some of the images are intentional, e.g. `lux-algo` which was what I built. I chose the first doc page instead of the landing page because our landing page is not polished enough. I'm going to pick those that are obviously outdated

**Comment by Josh-Cena** [comment]
Also @PatelN123 is it a screen size thing? None of the screenshots you submitted have sufficient height to fill the container.

image

**Comment by PatelN123** [comment]
I'll go through them again, when I have time.

**Comment by Josh-Cena** [comment]
Thanks! ❤️ It's still a truly great contribution. We can't thank more for anyone doing this chore

**Comment by PatelN123** [comment]
Thank you 😄

File: website/src/data/users.tsx (hunk 4 above)
Comment (vandreleal) [reply website/src/data/users.tsx]: I've updated the website link in the following PR https://github.com/facebook/docusaurus/pull/6242. Thanks, @PatelN123.

**Comment by PatelN123** [comment]
Just checking if I have now got the correct screen resolution.

**Comment by Josh-Cena** [comment]
Yes—that's much better! Thanks! We can port this aspect ratio to the rest

**Comment by PatelN123** [comment]
@Josh-Cena Can you please now confirm that the screen size is correct?
I have re uploaded the image for `agilets` here: [agilets](https://deploy-preview-6231--docusaurus-2.netlify.app/showcase/#:~:text=All%20sites-,AgileTs,-source)

Looks fine on my screen, I have tried zooming in and out, and looks good to me. 

I just want a second approval, so I know if I should continue how I am!

**Comment by PatelN123** [comment]
> Yes—that's much better! Thanks! We can port this aspect ratio to the rest

Awesome! I'll get that done ASAP

**Comment by slorber** [comment]
Thanks for your contribution

I'm not sure how to review it easily as GH does not show image diffs.

@Josh-Cena if you reviewed and think it's good let's merge 👍 it's just our site so it won't be harmful to any user apart ourselves anyway

**Comment by Josh-Cena** [comment]
> I'm not sure how to review it easily as GH does not show image diffs.

There's a button that displays a "rich diff" next to the image's title. It's a bit slow though

Not quite ready yet, waiting on @PatelN123 to change the aspect ratio so the images fill the container—he said to be back on Thursday. I think our image ratio test probably wasn't set exactly correct.

**Comment by slorber** [comment]
> There's a button that displays a "rich diff" next to the image's title. It's a bit slow though

Oh great didn't know! that's convenient!

> Not quite ready yet, waiting on @PatelN123 to change the aspect ratio so the images fill the container—he said to be back on Thursday. I think our image ratio test probably wasn't set exactly correct.

The test accept images that do not exactly match the card image ratio, because we have some existing bad ratios to fix.

If you want to fix those bad ratio images, we can fix the test and require the exact ratio that we need so that every card renders fine. After resizing we could also define a max-width or something to ensure we only accept smaller images.

I was wondering if we shouldn't create a GitHub discussion for site owners that want to be added.

IMHO it could be faster if regular contributors and maintainers do this themselves in batches, instead of having one PR per site? 🤷‍♂️ 

As we apply image constraits, this will lead a lot of noise as PRs will fail due to CI checks

**Comment by Josh-Cena** [comment]
> I was wondering if we shouldn't create a GitHub discussion for site owners that want to be added.
>
> IMHO it could be faster if regular contributors and maintainers do this themselves in batches, instead of having one PR per site? 🤷‍♂️

I once had the idea that we should have a form like Google Forms that people fill out and we can review on our side, and the data would be stored in some cloud instead of git... This means no changelog / git history noise, less review burden, don't have to maintain a 2000-line data file, and smaller repo size (the image folder is huge now)

> If you want to fix those bad ratio images, we can fix the test and require the exact ratio that we need so that every card renders fine. After resizing we could also define a max-width or something to ensure we only accept smaller images.

I think we can do the resizing first and then come back to this PR, okay @PatelN123? You can wait for some time before we merge #6043

**Comment by PatelN123** [comment]
Yeah, I will get the images done as soon as possible.

**Comment by Josh-Cena** [comment]
Thank you so much @PatelN123! Open source is great exactly because of contributors like you😄

**Pull request merged**